class-udm-updater.php

```diff
@@ -515,7 +515,7 @@ public function admin_notice_not_connected() {
 		echo '</div>';
 		echo "<script>
 		jQuery(function() {
-			jQuery('#udmupdater_not_connected').appendTo('.wrap p:first');
+			jQuery('#udmupdater_not_connected').appendTo('.wrap p').first();
```

Comment:
@andtown The `.first()` selector on the end there is doing nothing. It can't work backwards so that the `appendTo()` applies to what follows it.

Reply:
@andtown i.e. The new line is doing an `appendTo()` on all elements matching `.wrap p`, not just the first.

Reply:
@DavidAnderson684 Ahh yes, thanks for spotting that one. However, with the `ready` event that the code was in, I don't think the code would even work with it, I've tested it but nothing happened on my end. So, I've replaced it with `onload` event to execute the code once the page is fully loaded. Also, there was a style issue when moving the `#udmupdater_not_connected` element to the first `p` element, it didn't look good on the screen, many elements are misaligned and needed to be adjusted, so I've changed the way it appends the `#udmupdater_not_connected`, instead of appending the `#udmupdater_not_connected`  to the first `p` element, the `#udmupdater_not_connected` is now appended to the first `form` element and it only need just one inline css.

Here's how the UI looks like when the `ready` event was used to execute the code:

![image](https://user-images.githubusercontent.com/400912/103368202-927e6080-4af9-11eb-9046-c43db4ec4002.png)

Here's how the UI looks like if we append `#udmupdater_not_connected` to the first `p` element with the `onload` event

![image](https://user-images.githubusercontent.com/400912/103368843-34527d00-4afb-11eb-95cb-992948caff34.png)

and with the current changes I've just made, here's how the UI looks now:

![image](https://user-images.githubusercontent.com/400912/103369127-e8540800-4afb-11eb-8282-305172c0d5c8.png)